pkgs/development/interpreters/nextflow/default.nix

```diff
@@ -24,12 +24,6 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [
     makeWrapper
-    openjdk17
-    wget
-    which
-    gnused
-    gawk
-    coreutils
```

Comment:
Okay, thanks for this @SFrijters . If I change my PR (which I'm happy to do), this will clash. Could you rather not change the nextflow derivation in your PR and simply wait till mine is merged?

Reply:
Sure, I just put this up as a draft to get your opinion, we can proceed (or not) however you want.

This just fixes eval since openjdk doesn't cross-compile and I don't actually use this package myself (it's just a cleanup) so I would like you to check/review that it doesn't break anything for normal use when you're ready.

Reply:
Thanks for the reminder, I've rebased so this PR now again only affects `nf-test` itself.